src/js/components/List/stories/Order.js

```diff
@@ -14,7 +14,7 @@ export const Order = () => {
   const [ordered, setOrder] = useState(locations);
   return (
     <Box align="center" pad="large">
-      <List data={ordered} onOrder={setOrder} />
+      <List aria-label="list" data={ordered} onOrder={setOrder} />
```

Comment:
```suggestion
      <List aria-label="selection list" data={ordered} onOrder={setOrder} />
```
Can we change the aria-labels to be a little more descriptive so that we are not only meeting accessibility requirements but also providing the best experience for screenreader users? An aria-label of list doesn't give users a lot of information. Maybe we can change the aria-label to 'selection list'?